### ISSUE TITLE
Remove node_modules from repository

### ISSUE THREAD

**Issue by vitaminwater** [HackerLoop/pageblanche#4]
I'm afraid the only way to fix this is to restart the git repository from scratch 👉👌

**Comment by raphisky** [comment]
noooooooooooooooes :D

**Comment by squirelo** [comment]
tried a git -r --cached on nodes modules. I assume it has worked pretty well, but can you confirm @vitaminwater? thx

**Comment by vitaminwater** [comment]
No, this only removes node_modules from further commits, but it is still present in the previous commits. thus still there somewhere in the .git for eternity.

More seriously, I remember that there is a way to remove something from ALL commits (even previous), but it requires to tell everyone to `rm` and re-`clone` the repository.

**Comment by squirelo** [comment]
So the question is: Could @raphisky handles such a legacy in his carreer? 🤡
